DSL/com.larsreimann.safeds/src/test/resources/schema/testDataset.csv

```diff
@@ -0,0 +1,150 @@
+PassengerId,Survived,Pclass,Name,Sex,Age,SibSp,Parch,Ticket,Fare,Cabin,Embarked
```

Comment:
Where is this dataset from? How is it licensed? I guess it's the titanic dataset, right?

Reply:
Yes, titanic dataset.